lib/chef/resource/aws_rds_subnet_group.rb

```diff
@@ -0,0 +1,20 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+require 'chef/resource/aws_subnet'
+
+class Chef::Resource::AwsRdsSubnetGroup < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS
+
+  attribute :db_subnet_group_name, kind_of: String, name_attribute: true
+  attribute :db_subnet_group_description, kind_of: String
```

Comment:
These attributes should just be `name` and `description` - we already know we are within the RDS Subnet Group object